centreon/www/install/php/Update-next.php

```diff
@@ -174,10 +186,10 @@
 };
 
 try {
-    // TODO add your function calls to update the real time database structure here
+    $createJwtTable($peadDB);
```

Comment:
```suggestion
    $createJwtTable();
```
$pearDB is used in the function, no need to add it here